tests/test_parser.py

```diff
@@ -270,6 +270,36 @@ def test_default_order_used_if_date_order_not_supplied(self, date_string, expect
         self.then_date_exactly_is(expected_date)
         self.then_period_exactly_is(expected_period)
 
+
+    @parameterized.expand([
+        param(
+            date_string=u"20110101",
+            expected_date=datetime(2011, 1, 1),
+            expected_period='day',
+        ),
+        param(
+            date_string=u"01201702",
+            expected_date=datetime(1702, 1, 20),
+            expected_period='day',
+        ),
+        param(
+            date_string=u"01202020",
+            expected_date=datetime(2020, 1, 20),
+            expected_period='day',
+        ),
+        param(
+            date_string=u"20202001",
+            expected_date=datetime(2020, 1, 20),
+            expected_period='day',
+        ),
+    ])
```

Comment:
See if you can add a test that results in `None`, to cover that scenario and complete [test coverage on the changes](https://codecov.io/gh/scrapinghub/dateparser/pull/639/diff#D1-156).

Reply:
Sure added.